Library/Homebrew/tap.rb

```diff
@@ -545,6 +546,17 @@ def tap_migrations
     end
   end
 
+  # Hash with audit exceptions
+  def audit_exceptions
+    require "json"
```

Comment:
Pretty sure this is already required. If not: can be at the top of the file instead.